javascript/javascript1/week1/freeCodeCampUser.txt

```diff
@@ -0,0 +1,2 @@
+@fcc1b78ebba-c912-4448-8f74-4bb7f9336c68
+Florentina Petica
```

Comment:
Don't need to add this here. free code camp only for practice and to get an idea about JS Structure.